unic/ucd/normal/src/gen_cat.rs

```diff
@@ -13,12 +13,10 @@
 mod mark {
     use std::cmp::Ordering;
 
-    const GENERAL_CATEGORY_MARK: &'static [(char, char)] =
-        include!("tables/general_category_mark.rsv");
-
     /// Return whether the given character is a combining mark (`General_Category=Mark`)
     pub fn is_combining_mark(c: char) -> bool {
-        GENERAL_CATEGORY_MARK.find(c).is_some()
+        const TABLE: &'static [(char, char)] = include!("tables/general_category_mark.rsv");
+        TABLE.find(c).is_some()
```

Comment:
Wait how is this working when you aren't `use`ing `unic_utils::CharDataTable` anywhere in the file?